docker/run.sh

```diff
@@ -28,16 +33,15 @@ cd $PVC_MOUNT_PATH
 echo "exporting uac_qid_link table content (no UACs)"
 
 
-
-QID_FILE=qid_$(date +"%Y-%m-%dT%H-%M-%S").json
+QID_FILE=qid_$PERIOD_DATE.json
 
 PGPASSWORD=$DB_PASSWORD psql "sslmode=verify-ca sslrootcert=/root/.postgresql/root.crt sslcert=/root/.postgresql/postgresql.crt sslkey=/tmp/client-key.pem hostaddr=$DB_HOST port=$DB_PORT user=$DB_USERNAME dbname=rm" \
--c "\copy (SELECT row_to_json(t) FROM (SELECT id,qid,caze_case_ref as case_ref FROM casev2.uac_qid_link) t) To '$QID_FILE';"
+-c "\copy (SELECT row_to_json(t) FROM (SELECT id,qid,caze_case_id as case_id FROM casev2.uac_qid_link where last_updated >= '$START_OF_PERIOD' and last_updated < '$END_OF_PERIOD') t) To '$QID_FILE';"
```

Comment:
I would want to check with @zekizeki that it's OK to change the output of this file from case refs to case IDs... there will be a downstream impact on DAP so we should at least give them a heads up about this change.

Reply:
thanks @NickGrantONS MI team are expecting some changes so they'll be fine. I've let processing know as well.